.github/container/create-distribution.sh

```diff
@@ -78,18 +95,36 @@ if [[ $# -ge 1 ]]; then
     usage 1
 fi
 
-set -euox pipefail
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+set -eou pipefail
+# readlink -f $(pwd) is cross-platform way to ensure /tmp gets resolved correctly on macos
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && readlink -f $(pwd) )
 
-INSTALLED_DIR=${INSTALLED_DIR:-/opt/t5x}
-DISTRIBUTION_BASE_REF=${DISTRIBUTION_BASE_REF:-HEAD}
-MIRROR_GIT_URL=${MIRROR_GIT_URL:-https://github.com/nvjax-svc-0/t5x.git}
-
-if [[ -z "${INSTALLED_DIR}" ]]; then
-  echo "[ERROR]: Need to specify -d/--dir"
+if [[ -z "$MANIFEST" || -z "$PACKAGE" ]]; then
+  echo "--manifest and --package must be provided"
   usage 1
 fi
 
+BASE_DIR=${BASE_DIR:-/opt}
```

Comment:
I don't see where you set BASE_DIR before this line. Looks like BASE_DIR is always `/opt`. Or `BASE_DIR` is env variable?

Reply:
Good catch. It's not used anymore. Will remove